pkg/store/proxy.go

```diff
@@ -591,22 +597,30 @@ func storeMatches(ctx context.Context, s Client, mint, maxt int64, matchers ...*
 
 	storeMinTime, storeMaxTime := s.TimeRange()
 	if mint > storeMaxTime || maxt < storeMinTime {
-		return false, fmt.Sprintf("does not have data within this time period: [%v,%v]. Store time ranges: [%v,%v]", mint, maxt, storeMinTime, storeMaxTime)
+		s := "does not have data within this time period"
```

Comment:
We can just make it a const

Reply:
Sure! You mean inside the function `const s string = "..."`, or package-level?

Reply:
I think it should be fine as local const. Since we are not using anywhere else.

Reply:
Done ✅